bpf/lib/host_firewall.h

```diff
@@ -16,417 +16,417 @@
 
 # ifdef ENABLE_IPV6
 static __always_inline int
-ipv6_host_policy_egress(struct __ctx_buff *ctx, __u32 src_id,
```

Comment:
This diff is huge because tabs were replaced by spaces (in other places too). Please, push a new version

Reply:
Ugh yeah, wrong editor profile. Will fix.